src/app/index.ts

```diff
@@ -199,20 +189,29 @@ export class Auth {
  updateUserMenu() {
     if (this.authToken) {
       this.getUser(this.authToken, (response: any) => {
-        let user = response.data;
-        if (user.attributes.imageURL) {
-          $("#userImage")
-            .attr("src", user.attributes.imageURL)
-            .removeClass("hidden");
+        // check to see if user is on root and logged in successfully - if so, move them to gettingstarted
+        if (window.location.pathname.indexOf('get-involved') === -1 &&
+            window.location.pathname.indexOf('features')  === -1) {
```

Comment:
We now have a PDF at https://openshift.io/osiogettingstarted.pdf 

Is this IF to brittle? We will have to change this for each new page added to the landing page.

Reply:
the pdf won't matter - this isn't a webserver, it's just the html loading the index.js - but yes, we will need to adjust this to ignore each page